### ISSUE TITLE
prefer_inline: false positive when using interface/class?

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#2207]
When using global interface and class, it says to use inline declaration. Is there a way to do this in ABAP and avoid the error? 

https://github.com/mbtools/TEST_INLINE/runs/3920601749

![image](https://user-images.githubusercontent.com/59966492/137645638-9d9c2a78-0f04-4d53-b29c-336b27f89475.png)

Program `ZTEST_INLINE_GLOBAL`:

```abap
  DATA li_test TYPE REF TO zif_test_inline.
  li_test = NEW zcl_test_inline( ).
  li_test->test_1( ).
```

Interestingly, if class/interface are defined locally in the program, as expected, there's no error (see `ZTEST_INLINE_LOCAL`).

**Comment by mbtools** [comment]
ref #2250

**Comment by larshp** [comment]
yea, this is also a bug in https://github.com/abapGit/upport
